blindml/frontend/config/task/task.py

```diff
@@ -101,38 +105,35 @@ def __init__(self, task_fp) -> None:
         )
 
     def search_for_model(self):
-        # this will resume? if experiment already exists?
-        run_nni(self._nni_experiment_config)
-
-    def get_model_search_update(self):
-        sorted_good_trials = get_experiment_update(self._nni_experiment_config)
-        # re-sort by time
-        metric_values = [s["finalMetricData"] for s in sorted_good_trials]
-        if len(metric_values) == 0:
-            print("no successfully trained models yet")
-        return metric_values[::-1]
-
-    def get_best_model(self):
-        sorted_good_trials = get_experiment_update(self._nni_experiment_config)
-        hyper_parameters = sorted_good_trials[0]["hyperParameters"]
-        model = get_model(hyper_parameters)
-        return model
-
-    def train_best_model(self):
-        model = self.get_best_model()
+        # this only does regressions. choice of model is defined in
+        # the task defininition, though
+        # TODO: pick model type from there from there
 
-        # # TODO: this should be part of model search
-        # X_scaled = scale(X)
+        regressor = autosklearn.regression.AutoSklearnRegressor(time_left_for_this_task = self.task_search_time)
 
+        print("getting training data")
         X_train, y_train = self._data_set.get_train_data()
-        feat_idxs = select_features(X_train, y_train)
-        X_selected_train = X_train[:, feat_idxs]
 
-        return train(X_selected_train, y_train, model)
+        print("starting regressions")
+
+        # TODO: should give this both training and test data?
+        # which Task has already split up
+        # or give it all data and let regressor split it all
+        # up itself?
```

Comment:
[their example indicates that autosklearn does not split up the data](https://automl.github.io/auto-sklearn/master/#example)

```
    X_train, X_test, y_train, y_test = \
            sklearn.model_selection.train_test_split(X, y, random_state=1)
    automl = autosklearn.classification.AutoSklearnClassifier()
    automl.fit(X_train, y_train)
```

so i think the answer is no here